FHEM/00_SIGNALduino.pm

```diff
@@ -4638,11 +4638,11 @@ sub SetRegisters  {
 	
 	## check for four hex digits
 	my @nonHex = grep (!/^[0-9A-Fa-f]{4}$/,@a[1..$#a]) ;
-	return "ERROR: wrong parameter value @nonHex, only hexadecimal ​​four digits allowed" if (@nonHex);
+	return "$hash->{NAME} ERROR: wrong parameter value @nonHex, only hexadecimal ​​four digits allowed" if (@nonHex);
```

Comment:
@elektron-bbs,
wenn du die Returnwerte änderst, das sind die Werte welche du dann im Test anpassen musst sicherlich.

Diese Ausgaben wurden fix genommen von @sidey79 zur Verifizierung.

Reply:
sorry, was besseres ist mir vermutlich nicht eingefallen :)

Reply:
Hab ich mir schon gedacht. Bin mir nur noch nicht sicher, ob ich die richtige Stelle gefunden habe:
test_sub_SIGNALduino_Set-definition.txt
Zeile 381
`		    return =>  match qr/^ERROR: wrong parameter value AP23/
`
ändern in
`		    return =>  match qr/ERROR: wrong parameter value AP23/
`
???

Reply:
Bei Unsicherheit nutze das TOOL um dir den Test herunter zu laden und so kannst du es vor dem hochladen testen :-)

Reply:
Ja, mit der Anpassung würde es gehen :)